caesars-palace/blocks/featured-card/featured-card.js

```diff
@@ -0,0 +1,18 @@
+export default function decorate(block) {
+  const blockElements = [...block.children];
+  const backgroundImageElement = blockElements[0];
+  backgroundImageElement.classList.add('featured-card-background');
+
+  const blockContent = blockElements[1];
+  blockContent.classList.add('featured-card-content');
+
+  blockContent.querySelectorAll('img').forEach((image) => {
+    image.closest('div').classList.add('featured-card-image');
+  });
+
+  [...blockContent.children].forEach((item) => {
+    if (![...item.classList].includes('featured-card-image')) {
```

Comment:
`item.classList.contains('featured-card-image')` should acheive the same result without having to use array functions.

https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList/contains